index.js

```diff
@@ -16,6 +16,54 @@ import apiKeyAuth from './api-key-auth';
 import logger from './logger';
 import version from './version';
 
+// Variable that saves when a tx is being sent
+// We don't support sending multiples transactions concurrently
+// so we use this to prevent a user from sending multiples requests
+// while the first was not finished
+let sendingTx = false;
```

Comment:
The implementation seems good to me. Just a few comments:

1) Rename variables and methods to a lock system. So, `sendingTx` would become `lock_status`, and `startSendTx` would become `getLock({timeout=SEND_TX_LOCK_TIMEOUT})`.

2) Add a comment that the lock is local and only works with one thread and no balance between multiple instances. In those cases, we would need to add support for a distributed lock (e.g. using redis).

Reply:
Done! Thanks!